src/posts/us-exit-documents/index.md

```diff
@@ -0,0 +1,307 @@
+but OnlyOffice and Microsoft Office (requires an Enterprise license and an available Office Online Server) can be set up as editors instead.
+I'm not a huge fan of the default Nextcloud Office based on Collabora.
+I like that Collabora is built on [LibreOffice](https://www.libreoffice.org/), but the UI is low on contrast and difficult to navigate.
+If we were to pick Nextcloud, I'd probably opt for replacing Nextcloud Office with OnlyOffice.
+
+As for running Nextcloud, I could easily set it up in a VM on my home server and use it over VPN.
+While pretty much all of us at factor10 could run our own instance of Nextcloud on a public cloud provider, that is not something we want to do. We'd rather focus on our customers and buy the hosting from someone else,
+luckily for us, there are several options available.
+
+I'm not going to go into a detailed cost analysis here, but self-hosting Nextcloud on a public cloud runs anywhere from about EUR 450 to EUR 200 a month for a server that can store 500 GiB of data to 20 users.
+
+Paid-for hosting of Nextcloud is available from several providers and the ones I've looked up are:
+
+* [RedPill-Linpro](https://www.redpill-linpro.com/en) - A Swedish cloud/SaaS provider that specialises in hosting open source software
+* [Safekloud](https://safekloud.eu) - Hosted Nextcloud from the Swedish webhost WebbPlatsen.
+* [ITSL](https://itsl.se/) - The hosting branch of IT-Säkerhetsbolaget, a Swedish consultancy focused on data security.
+* [NetWays](https://nws.netways.de/en/saas/nextcloud/) - A German IT company that provides consulting services as well as hosting of a lot of open source software
+* [Stellar](https://www.stellarhosted.com/nextcloud/) - A Dutch company that hosts open source software
+
+Prices and configuration of these differ a lot and some are Nextcloud Enterprise while some are not.
+If you're interested in hosted Nextcloud I urge you to check out the above and talk to their sales departments about price.
+
+## Jottacloud
+
+[Jottacloud](https://jottacloud.com/en/) is a Norwegian company focused on cloud storage.
+They offer various plans for both persons and business and have a strong focus on backing up your devices.
+
+For document editing, they use Microsoft Office Online, which sends your documents to Microsoft's servers while editing and then syncs the changes back to Jottacloud.
+This makes them a no-go for factor10, unfortunately.
+
+Syncing on Linux is done via their own CLI application and as far as I can tell, it's immediate and not scheduled.
+
+They offer various plans for both personal and business use.
+The one that would suit me is SEK 149 (about EUR 13.5) for 5 TiB of storage with up to five users.
+What that offer doesn't say however is that those 5 TiB is per user, so each user gets 1 TiB of storage and you can't use more than that on each account.
+
+## pCloud
```

Comment:
It just occurred to me that it would be great with a section about Storegate too (at least for me personally since I know the founders). :)

They haven been a driving force in this space for years, but I think they focus on storage rather than collab...?

Reply:
De skriver att de har collab via MS Office, vilket antingen innebär att de skickar filerna till MS servrar eller att man behöver Office Online Server någonstans i närheten (vilket de kanske står för iofs).